packages/data/src/types/abstract-query/fields/nested/relational.ts

```diff
@@ -41,37 +48,38 @@ export interface AbstractQueryFieldNodeRelationalOneToAny {
  * @example
  * ```
  * const functionNode = {
- * 	current: {
- * 		fields: ['id']
- *  },
- * 	external: {
+ * 	local: {
+ * 		fields: ['id'],
+ *    },
+ * 	foreign: {
  * 		store: 'mongodb',
  * 		collection: 'some-collection',
- * }
+ *    },
+ * };
  * ```
  */
 export interface AbstractQueryFieldNodeRelationalJoinMany {
 	/** the fields of the current collection which have the relational value to an external collection or item */
-	current: {
-		fields: [string, ...string[]];
+	local: {
+		fields: AtLeastOneElement<string>;
 	};
 
 	/** the external collection or item which should be pulled/joined/merged into the current collection */
-	external: {
-		store?: string;
+	foreign: {
+		store: string;
```

Comment:
Was `store` intentionally made required?

Reply:
yes. how do we know otherwise, which collection the user wants to be "joined"?

Reply:
yes. how do we otherwise know where the collection lives in?

Reply:
If `store` is not defined, it will be in the same store as the root query. But I'd be fine with making it explicit.

Reply:
being explicit here would make it a bit easier so let's keep it like this. similarly with the alias actually which we want to make required everywhere